source/simulator/helper_functions.cc

```diff
@@ -1484,6 +1484,212 @@ namespace aspect
 
 
   template <int dim>
+  void Simulator<dim>::compute_reactions ()
+  {
+    if (time_step == 0)
+      return;
```

Comment:
`time_step` already is the time step length (the time step number is called `timestep_number`). I was confused about that myself several times already. I'll add a comment.